src/main/java/appeng/decorative/solid/BlockQuartzOre.java

```diff
@@ -76,16 +77,15 @@ public int quantityDropped( final Random rand )
 	}
 
 	@Override
-	public void dropBlockAsItemWithChance( final World w, final BlockPos pos, final IBlockState state, final float chance, final int fortune )
+	public int getExpDrop ( IBlockState state, IBlockAccess world, BlockPos pos, int fortune )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') '(' is preceded with whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.MethodParamPadCheck)